design-documents/graph-ql/coverage/bundle.graphqls

```diff
@@ -0,0 +1,30 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Mutation {
+    addBundleProductsToCart(
+        input: AddBundleProductsToCartInput,
+        check_inventory: Boolean @doc(description: "overrides the \"Check inventory on cart load\" admin setting")
```

Comment:
@cspruiell Can we add this check_inventory field inside AddBundleProductsToCartInput? Considering this is an optional field it will be backward compatible. Also the input types are created for the same reason.

Reply:
Same for all the other instances.